addon/components/bread-crumbs.js

```diff
@@ -28,6 +29,7 @@ export default Component.extend({
   reverse: false,
   classNameBindings: ['breadCrumbClass'],
   hasBlock: bool('template').readOnly(),
+  routing: inject.service('-routing'),
```

Comment:
Should this use the router service polyfill?

Reply:
Yes, it probably should now! I haven't upgraded to Ember 2.15 yet so I've only tested this PR against Ember versions up to 2.14.1 so far.

Reply:
Actually after posting this, I'm not sure the router service provides what is needed here...
The polyfill allows the use of the router service starting from 2.4